src/framework/ui/uitypes.h

```diff
@@ -54,6 +54,18 @@ struct ContainerMeta
     ContainerMeta(const ContainerType::Type& type, int widgetMetaTypeId)
         : type(type), widgetMetaTypeId(widgetMetaTypeId) {}
 };
+
+class Border : public QObject
```

Comment:
I believe uicomponents module is a more suitable place for this

Reply:
I'm more interested in why you decided to move the border to c++?

Reply:
Fixed, thanks!
Actually, it just didn't came to my mind to put it in QML 😊, but that's indeed a good idea!